src/ucp/core/ucp_request.h

```diff
@@ -46,6 +46,18 @@ enum {
 
 
 /**
+ * Protocols enumerator to work with send request state
+ */
+enum {
+    UCP_REQUEST_SEND_STATE_SHORT = 0,
```

Comment:
what is the goal here?
what happens for combinations? e.g SHORT+PUT?

Reply:
The goal is to encapsulate and unify changing of the request state (data type + uct_comp) into single function for all protocols. What do you mean "for combination"? these are not flags, just enumerator. If you see some usage for combination - pls let me know. I'll remove SHORT (not needed) and replace PUT and GET with RMA.